sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLLibraryDAO.java

```diff
@@ -398,7 +398,7 @@ public long save(Library library) throws IOException {
 
           String name = namingScheme.generateNameFor(library);
           library.setName(name);
-          DbUtils.validateNameOrThrow(library, namingScheme);
+          NamingScheme.validateNameOrThrow(library, namingScheme);
```

Comment:
The `DbUtils` class isn't accessible from `MisoRequestManager`, which lives in core, so I had to relocate this method. It's also not a DAO-specific method; it's part of naming schemes, which live in core anyway.

There's no sane case when the DAOs have different naming schemes wired in (I hope).